pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -463,7 +373,6 @@ def build_package(path: Path, args):
             # i.e. they need package running, but not compile
             if not pkg.get("build", {}).get("sharedlibrary"):
                 compile(path, srcpath, pkg, args, bash_runner)
-            package_files(buildpath, srcpath, pkg, compress=args.compress_package)
```

Comment:
Currently this is handled in buildall:
```py
        if self.shared_library:
            file_path = shutil.make_archive(
                f"{self.name}-{self.version}", "zip", self.pkgdir / "dist"
            )
            shutil.copy(file_path, outputdir)
            return
```
Maybe we should zip the file in buildpkg instead though? And of course we could use a tar file instead of a zip file if you prefer.